### DISCUSSION TITLE
What are any related data analysis pipelines that you or your group have developed?

### DISCUSSION THREAD

**Discussion by sr320** [sr320/ceabigr#8] [General]
These could be partial in nature and certainly do not have to perfected in nature.

**Comment by sr320** [comment]
our DNA methylation workflow https://robertslab.github.io/resources/bio_DNA-methylation/

**Comment by shaylematsuda** [comment]
What I use for 16S - developed at UH with Craig Nelson's group: [https://metagenomics-pipelines.readthedocs.io/en/latest/index.html]
